webapp/src/components/buttons/primaryButton/index.tsx

```diff
@@ -5,12 +5,15 @@ type PrimaryButtonProps = {
     iconName?: string
     extraClass?: string
     onClick?: () => void
+    isSecondaryButton?: boolean
```

Comment:
If this button is getting such a prop for `isSecondaryButton`, then the folder in which this file is present should be renamed. The folder is `primaryButton`, it should be changed to something like `genericButton` or something else.